### DISCUSSION TITLE
Trilium & TriliumNext

### DISCUSSION THREAD

**Discussion by neilcrew** [tteck/Proxmox#3536] [General] [answered]
Are there any plans to shift the Trilium script to use TriliumNext, as this now seems to be the vehicle for continued development?

**Comment by tteck** [comment, marked as answer]
A quick search will reveal the answer. https://github.com/tteck/Proxmox/discussions/408#discussioncomment-10277294

**Comment by tteck** [comment]
Or, a code inspection https://github.com/tteck/Proxmox/blob/main/install/trilium-install.sh

**Comment by tjeerdhans** [comment]
Might I suggest to change the link in the description at https://tteck.github.io/Proxmox/#trilium-lxc to the TriliumNext repo? And maybe also change the name to TriliumNext (not sure about that, though..)?
